### PR TITLE
Add amount of friends and connections

### PR DESCRIPTION
**Pull request by oSumAtrIX**
RE: #8 

- [x]  Amount of friends
- [ ]  Amount of guilds
- [ ]  Owned guilds
- [ ]  Current status (online, offline ... custom status)
- [x]  Connections

This PR adds connections and amount of friends. The type of connection will be lowercase, for that, a dictionary should be made to match against the lowercase names

### CODE REVIEW

File: DTI.py [hunk 1]
```diff
@@ -8,34 +8,34 @@
...
+    'cs': 'Czech, Czechia, Czech Republic',
+    'el': 'Greek, Greece',
+    'bg': 'Bulgarian, Bulgaria',
+    'ru': 'Russian, Russia',
+    'uk': 'Ukranian, Ukraine',
+    'th': 'Thai, Thailand',
+    'zh-CN': 'Chinese, China',
+    'ja': 'Japanese',
+    'zh-TW': 'Chinese, Taiwan',
+    'ko': 'Korean, Korea'
```
Comment (Omicron166) [inline DTI.py]: why? The old format looks better and it's easier to read

File: DTI.py [hunk 2]
```diff
@@ -153,25 +159,40 @@ def main():
...
+                print(
+                    f'    {Fore.RESET}Username               {Fore.GREEN}{user_name}')
+                print(
+                    f'    {Fore.RESET}User ID                {Fore.GREEN}{user_id}')
+                print(
+                    f'    {Fore.RESET}Creation Date          {Fore.GREEN}{creation_date}')
+                print(
+                    f'    {Fore.RESET}Avatar URL             {Fore.GREEN}{avatar_url if avatar_id else ""}')
+                print(
+                    f'    {Fore.RESET}Token                  {Fore.GREEN}{token}')
```
Comment (Omicron166) [inline DTI.py]: Maybe you're taking the formatting TOO seriously?
The old format is even easier to read

File: DTI.py [hunk 3]
```diff
@@ -153,25 +159,40 @@ def main():
...
+                    f'    {Fore.RESET}Phone Number           {Fore.YELLOW}{phone_number if phone_number else ""}')
+                print(
+                    f'    {Fore.RESET}Email                  {Fore.YELLOW}{email if email else ""}')
+                print(f'{Fore.RESET}\n')
+
+                print('Connections')
+                print('-------------------')
+                for connection in requests.get(f'https://discordapp.com/api/v6/users/{user_id}/profile?with_mutual_guilds=false', headers=headers).json()['connected_accounts']:
+                    print('    {:<23}{}{}'.format(
+                        connection['type'], Fore.CYAN, connection['name']))
```
Comment (Omicron166) [inline DTI.py]: Interesting feature, seeems good

**Review by Omicron166 — APPROVED** [review]
Calm down with the code format, anyways, nice new features

File: DTI.py (hunk 2 above)
Comment (oSumAtrIX) [reply DTI.py]: Reverted in commit bf3b2c402fbcfbf596999b1e01466e9eb09e8fba

File: DTI.py (hunk 1 above)
Comment (oSumAtrIX) [reply DTI.py]: Reverted in commit bf3b2c402fbcfbf596999b1e01466e9eb09e8fba

**Review by Omicron166 — APPROVED** [review]
Nice, this should be merged as soon as possible.

**Pull request closed without merging**